packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -145,15 +145,35 @@ export interface FunctionProps {
 
   /**
    * What security group to associate with the Lambda's network interfaces.
+   * This property is being deprecated, consider using securityGroups instead.
```

Comment:
lang: `This property is deprecated, use securityGroups instead`

Reply:
Done.